### PR TITLE
Enable the login rate limit

### PR DESCRIPTION
**Pull request by bytehead**
Implements https://github.com/contao/contao/issues/3821#issuecomment-1008708462

Enabling the the [DefaultLoginRateLimiter](https://github.com/symfony/symfony/blob/5.4/src/Symfony/Component/Security/Http/RateLimiter/DefaultLoginRateLimiter.php)  using the [Fixed Window Rate Limiter](https://symfony.com/doc/current/rate_limiter.html#fixed-window-rate-limiter) policy.

### CODE REVIEW

File: manager-bundle/skeleton/config/security.yaml [hunk 1]
```diff
@@ -19,6 +19,10 @@ security:
             contao_login:
                 remember_me: false
 
+            login_throttling:
+                max_attempts: 5
+                interval: 1 minute
```
Comment (ausi) [inline manager-bundle/skeleton/config/security.yaml]:
```suggestion
            login_throttling: ~
```

According to the docs this should use the default of 5 per minute. ☺️

Comment (bytehead) [reply manager-bundle/skeleton/config/security.yaml]: Correct, I thought to make it more obvious where to change what.

**Comment by ausi** [comment]
Shouldn’t we remove our own logic then so that these mechanisms don’t interfere with each other?
https://github.com/contao/contao/blob/09696f510b318cd94c594d9aa83a49d040c2a9f6/core-bundle/src/Security/User/UserChecker.php#L50

**Comment by bytehead** [comment]
Hmm, good question. Our mechanism works only for existing accounts so the Symfony rate limiter covers both (existing and non-existing).

**Comment by leofeyer** [comment]
Yes, we should remove our own logic IMHO.

**Review by leofeyer — APPROVED** [review]

**Comment by leofeyer** [comment]
Thank you @bytehead.

**Pull request merged**